charts/substra-backend/Chart.yaml

```
@@ -16,16 +16,20 @@ dependencies:
   - name: redis
     repository: https://charts.bitnami.com/bitnami
     condition: redis.enabled
-    version: 17.0.7
+    version: 18.1.2
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    condition: redis.enabled
```

If I'm reading this correctly: https://helm.sh/docs/topics/charts/#tags-and-condition-fields-in-dependencies

Then we could write this to avoid problems in case redis is disabled but another chart is (which should never happen but still):

```suggestion
    condition: redis.enabled,postgresql.enabled,minio.enabled
```